source/user-manual/api/configuration.rst

```diff
@@ -285,6 +285,8 @@ cache
 | time       | Any positive integer or real number  | 0.75          | Time in seconds that the cache lasts before expiring.                                                                |
 +------------+--------------------------------------+---------------+----------------------------------------------------------------------------------------------------------------------+
 
+.. deprecated:: 4.8.0
```

Comment:
I'd put it below the `cache` title since it's not clear if we are depecating that or `_api_configuration_access`.

Reply:
```
cache
^^^^^

.. deprecated:: 4.8.0

+------------+--------------------------------------+---------------+----------------------------------------------------------------------------------------------------------------------+
| Sub-fields | Allowed values                       | Default value | Description                                                                                                          |
+============+======================================+===============+======================================================================================================================+
| enabled    | yes, true, no, false                 | true          | Enable or disable caching for certain Wazuh API responses (currently, all :api-ref:`rules endpoints <tag/Rules>` )   |
+------------+--------------------------------------+---------------+----------------------------------------------------------------------------------------------------------------------+
| time       | Any positive integer or real number  | 0.75          | Time in seconds that the cache lasts before expiring.                                                                |
+------------+--------------------------------------+---------------+----------------------------------------------------------------------------------------------------------------------+
```

Reply:
Fixed in commit eac06b4a6df1b99eef2e1f05b5f975e0e33c09b6.